plugin.video.catchuptvandmore/resources/lib/resolver_proxy.py

```diff
@@ -300,7 +423,8 @@ def get_francetv_video_stream(plugin,
         final_video_url = json_parser2['url']
         if download_mode:
             return download.download_video(final_video_url)
-        return final_video_url + '|X-Forwarded-For=' + '2.' + str(randint(0, 15)) + '.' + str(randint(0, 255)) + '.' + str(randint(0, 255)) + '&User-Agent=' + web_utils.get_random_ua()
+        return final_video_url + '|X-Forwarded-For=' + '2.' + str(randint(0, 15)) + '.' + str(
```

Comment:
We don't want add-ons to circumvent GEO blocks, both in streaming and downloading. That means that using `x-forwarded-for` is not allowed.

Reply:
Thanks for the feedback. I removed the problematic code.